lib/ansible/modules/cloud/cloudstack/cs_network_offering.py

```diff
@@ -16,37 +16,44 @@
 description:
     - Create, update, enable, disable and remove network offerings.
 version_added: '2.5'
-author: "David Passante (@dpassante)"
+author: David Passante (@dpassante)
 options:
   state:
     description:
       - State of the network offering.
+    type: str
     choices: [ enabled, present, disabled, absent]
     default: present
   display_text:
     description:
       - Display text of the network offerings.
+    type: str
   guest_ip_type:
     description:
       - Guest type of the network offering.
+    type: str
     choices: [ Shared, Isolated ]
   name:
     description:
       - The name of the network offering.
+    type: str
     required: true
   supported_services:
     description:
       - Services supported by the network offering.
-      - One or more of the choices.
-    choices: [ Dns, PortForwarding, Dhcp, SourceNat, UserData, Firewall, StaticNat, Vpn, Lb ]
```

Comment:
I just wonder if we should add this choice list to the arg spec or remove it here. Any thoughts?

Reply:
I also wondered. I did not add the choice list to the arg spec to not be restrictive in the case new services would appear but I doubt that the list evolves a lot.
Finally, it may be better to add this in arg_spec.

Reply:
I'm going to wait for the PR #53901 to be merged before rollback this change to not fall into the validate-module issues.

Reply:
okay!